### ISSUE TITLE
Window controls overlap plugin/menu controls

### ISSUE THREAD

**Issue by jldeon** [MrOtherGuy/firefox-csshacks#172]
I'm running Firefox 97.0 on Windows 11.

My userChrome.css looks like:

```
@import url(window_control_placeholder_support.css);
@import url(hide_tabs_toolbar.css);

```

(where the imported files are the latest from this repository)

The right side of the navigation bar looks like:
![image](https://user-images.githubusercontent.com/4325406/153770127-a5c78f6b-5c00-47be-8f4b-6abfc9f1371f.png)

On the right side you can see that the Firefox buttons and the buttons added by plugins are overlapping the minimize/maximize/close buttons.

I thought perhaps I could make more space by adding:

```
:root[tabsintitlebar]{
  --uc-window-control-width: 600px; /* Space reserved for window controls (Win10) */
}
```
to userChrome.css, but this had no effect.

**Comment by MrOtherGuy** [comment]
Thanks for reporting, somehow a typo had ended up into `window_control_placeholder_support.css` in the last commit.

 Should be fixed with a5ac138

**Comment by jldeon** [comment]
Yep, that fixed it!  Thanks :)
